pyqtgraph/graphicsItems/CurvePoint.py

```diff
@@ -91,6 +91,9 @@ def paint(self, *args):
         pass
     
     def makeAnimation(self, prop='position', start=0.0, end=1.0, duration=10000, loop=1):
+        # automatic encoding when QByteString expected was removed in PyQt v5.5
+        if not isinstance(prop, bytes):
+            prop = prop.encode('latin-1')
```

Comment:
That's what PyQt was doing before they stopped automatically converting a `str` to `QByteArray`.

Looking more into this, I think this is probably not the right way to go, but expecting the user to explicitly pass a `bytes` type as a property name seems a little awkward. Here's a quote from the PyQt 5.4.2 doc strings (in `sip/QtCore/qbytearray.sip`):

```
// In Python v2.x we want a str object to be interpreted as the name of the
// file (as that is the historical behaviour).  This has the following
// implications.
//
// - The QString version of the ctor must appear before the QByteArray version
//   in the .sip file.  This rule should be applied wherever a similar
//   situation arises.
// - A QString must not automatically convert a QByteArray.
// - QByteArray must also exist in the v2 API.
//
// In Python v3.x we want a bytes object to be used wherever a QByteArray is
// expected.  This means that a QString must not automatically convert a bytes
// object.
//
// A QByteArray can be created from a Latin-1 encoded string.  This was a
// mistaken attempt to ease the porting of Python2 code to Python3.  It is
// deprecated in PyQt v5.4 and will be removed in v5.5.
```